docs/src/main/paradox/index.md

```diff
@@ -1,13 +1,16 @@
 # Alpakka
 
-Welcome to the home of the Alpakka innitiative, which harbours various Akka Streams connectors. Here you can find documentation of the connectors that are part of this project as well as links to connectors that are maintained by other projects.
+Welcome to the home of the Alpakka innitiative, which harbours various Akka Streams components for integration use cases. Here you can find documentation of the components that are part of this project as well as links to components that are maintained by other projects.
```

Comment:
@harksin You mean something like:

Welcome to the home of the Alpakka innitiative, which harbours various Akka Streams connectors, integration patters, and data transformations for integration use cases.